packages/api-client/src/components/DataTable/DataTableInputSelect.vue

```diff
@@ -98,9 +98,11 @@ watch(addingCustomValue, (newValue) => {
             :value="option"
             @click="updateSelected(option)">
             <div
-              class="flex items-center justify-center rounded-full p-[3px] w-4 h-4 group-hover/item:shadow-border"
+              class="flex items-center justify-center rounded-full p-[3px] w-4 h-4"
               :class="
-                isSelected(option) ? 'bg-blue text-b-1' : 'text-transparent'
+                isSelected(option)
+                  ? 'bg-c-accent text-b-1'
+                  : 'group-hover:shadow-border text-transparent'
```

Comment:
Is this missing `/item`?

```suggestion
                  : 'group-hover/item:shadow-border text-transparent'
```